Makefile

```diff
@@ -92,7 +92,8 @@ docker-build:
 
 docker-run:
 	@echo "++\n***** Running docker-compose\n++"
-	@docker-compose up
+	@yarn
+	@docker compose up --build
```

Comment:
While we are at it, I intentionally removed the build param because it is quite slow. Maybe it won't be an issue with your current setup. Ideally, I was thinking we can do another makefile command like `docker-up`  that runs 'docker-build' and `docker-run`.

Reply:
I like that idea!